messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -2117,6 +2123,56 @@ class MessagingTest : BaseMessagingTest() {
         }
     }
 
+    @Test
+    fun testRecovery() {
+        testInCoroutine {
```

Comment:
This only tests the happy path. Would be great to also test:
- when a recovery key does not work
- when a recovery key is empty